addons/binding/org.openhab.binding.km200/ESH-INF/thing/dhwCircuit.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="km200"
+		xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+		xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="dhwCircuit" listed="false">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="kmdevice" />
+		</supported-bridge-type-refs>
+		<label>Hot water circuit</label>
+		<description>This thing is representing a hot water circuit.</description>
+		<properties>
+			<property name="root">#dhwCircuits</property>
```

Comment:
@htreu what is this, is this a standard thing?

Reply:
No, its not. @Markinus do you really need this properties? The same is already defined in `KM200ServiceType`.

Reply:
Yes, you're right, I have to clean up here...